src/containers/BooksForm.js

```diff
@@ -28,18 +28,19 @@ const BooksForm = props => {
   };
   return (
     <>
-      <p>BooksForm</p>
-      <form>
-        <input placeholder="Title" name="bookTitle" onChange={handleChange} />
-        <select name="bookCategory" onChange={handleChange}>
+      <h2 className="add-new-book">ADD NEW BOOK</h2>
+      <form className="add-book-form">
+        <input className="input-book-title" placeholder="Book title" name="bookTitle" onChange={handleChange} />
+        <select className="select-category" name="bookCategory" onChange={handleChange}>
+          <option value="" disabled selected>Category</option>
           {categories.map(item => (
             <option key={item} value={item}>
               {item}
             </option>
           ))}
         </select>
-        <button type="button" value="submit" onClick={handleSubmit}>
-          Submit
+        <button className="submit-btn" type="button" value="submit" onClick={handleSubmit}>
+          ADD BOOK
         </button>
       </form>
     </>
```

Comment:
## Required Change
- [ ] We have some informing console errors during the creation of a book. React can't figure out how to create a book using default value on the category option, as of now. Please work on his changes too, before proceeding. Below is what the user is met with 👇 
![bookstore-design](https://user-images.githubusercontent.com/22345961/105897234-f88ed280-6028-11eb-8f72-9887e5fd850e.png)
 

## Optional Change
- [ ] Let's add a form validation so that a user cannot be allowed to add an empty book during book creation.
Currently, this is possible, though it is a bad practice. 👇 You can read more about this [here](https://www.educative.io/edpresso/react-form-validation).
![bookstore-implementation](https://user-images.githubusercontent.com/22345961/105896633-1c9de400-6028-11eb-8b10-05d5079ca41b.png)

Reply:
I've implemented form validation and fixed the issue with the select.